scripts/fire-time.sh

```diff
@@ -16,39 +17,54 @@
 #
 
 #
-# Ensure network is stable first then launch main-start
-#
+# Try keep system time in sync
 #
 
 : ${FIREWALLA_HOME:=/home/pi/firewalla}
 
-ntp_process_cnt=`sudo systemctl status ntp |grep 'active (running)' | wc -l`
-logger "FIREWALLA.FIRETIME.ENTER "+`date`
+logger "FIREWALLA.DATE.SYNC"
+
+TIME_THRESHOLD="2020-03-27"
+
+tsThreshold=$(date -d "$TIME_THRESHOLD" +%s)
+tsFakeHwclock=$(date -d "$(cat /etc/fake-hwclock.data)" +%s)
+if [ $tsFakeHwclock -ge $tsThreshold ]; then tsThreshold=$tsFakeHwclock; fi
 
 function sync_time() {
     time_website=$1
     logger "syncing time from ${time_website}..."
-    time=$(curl -m10 -D - ${time_website} -o /dev/null --silent | egrep "^Date:" | awk -F ": " '{print $2}')
+    time=$(curl -m10 -D - ${time_website} -o /dev/null --silent | awk -F ": " '/^Date: / {print $2}')
```

Comment:
if no bug found on the previous usage, don't change it..

Reply:
https://github.com/firewalla/firewalla/commit/bd688977294c02fe849df210849644c16440f29d